lib/bigcommerce/storefront-config.ts

```diff
@@ -43,3 +50,23 @@ const createCartRedirectUrl = () => {
 }
 
 export const memoizedCartRedirectUrl = createCartRedirectUrl();
+export const fetchStorefrontToken = async () => {
+  const response = await fetch(
+    `${BIGCOMMERCE_API_URL}/stores/${process.env.BIGCOMMERCE_STORE_HASH}/v3/storefront/api-token-customer-impersonation`,
+    {
+      method: 'POST',
+      headers: {
+        accept: 'application/json',
+        'content-type': 'application/json',
+        'x-auth-token': process.env.BIGCOMMERCE_ACCESS_TOKEN!,
+        'x-bc-customer-id': ''
+      },
+      body: JSON.stringify({
+        channel_id: parseInt(process.env.BIGCOMMERCE_CHANNEL_ID!),
+        expires_at: Math.floor(new Date().getTime() / 1000) + 1 * 24 * 60 * 60 // 1 day
+      })
+    }
+  );
+
+  return (await response.json()) as StorefrontTokenResponse;
+};
```

Comment:
@bc-alexsaiannyi I don't see where this function is used. Do you know why it was introduced here?

Reply:
it looks like it's been a  incorrect result of resolving conflicts. Thanks for catching this!